app/models/pusher.rb

```diff
@@ -60,10 +60,8 @@ def find
     @rubygem = Rubygem.name_is(name).first || Rubygem.new(name: name)
 
     unless @rubygem.new_record?
-      if @rubygem.find_version_from_spec(spec)
-        notify("Repushing of gem versions is not allowed.\n" \
-               "Please use `gem yank` to remove bad gem releases.", 409)
-
+      if (version = @rubygem.find_version_from_spec spec)
+        republish_notification version
```

Comment:
:thinking: `republish_notification(version)` :question:

Reply:
I did run rubocop before push. I guess we have disabled this: https://github.com/rubocop-hq/ruby-style-guide#method-invocation-parens

updated.

Reply:
ok, no worries than